tests/CMakeLists.txt

```diff
@@ -1,21 +1,20 @@
 set(CONFIGS
-    #"1024, 6200, uint64_t"
-	"8192, 124, uint64_t"
-    #"1024, 124, uint64_t"
-    #"1UL<<14, 620, uint64_t"
-    #"1024, 62, uint64_t"
-    #"1024, 37200, uint64_t"
-    "1024, 60, uint32_t"
-    #"1024, 3000, uint32_t"
-    "512, 30, uint32_t"
-	#"512, 14, uint16_t"
-	#"256, 14, uint16_t"
     "8, 60, uint32_t"
+    "128, 14, uint16_t"
+    "1024, 60, uint32_t"
+    "8192, 124, uint64_t"
```

Comment:
Well, we were not testing the 16-bit words - I think that was a mistake, so I replaced 512/30 by 128/14.